website_and_docs/content/blog/2023/whats-new-in-selenium-manager-with-selenium-4.11.0.md

```diff
@@ -0,0 +1,125 @@
+DEBUG   Driver URL: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/115.0.5790.102/win64/chromedriver-win64.zip
+INFO    Driver path: C:\Users\boni\.cache\selenium\chromedriver\win64\115.0.5790.102\chromedriver.exe
+INFO    Browser path: C:\Program Files\Google\Chrome\Application\chrome.exe
+```
+
+In this case, the local Chrome (in Windows) is detected by Selenium Manager. Then, using its version and the CfT endpoints, the proper chromedriver version (115, in this example) is downloaded to the local cache. Finally, Selenium Manager provides two results: i) the driver path (downloaded) and ii) the browser path (local).
+
+Let's consider another example. Now we want to use Chrome beta. Therefore, we invoke Selenium Manager specifying that version label as follows (notice that the CfT beta is discovered, downloaded, and stored in the local cache):
+
+```
+> selenium-manager --browser chrome --debug --browser-version beta
+DEBUG   Checking chromedriver in PATH
+DEBUG   Running command: chromedriver --version
+DEBUG   Output: ""
+DEBUG   chromedriver not found in PATH
+DEBUG   Checking chrome in PATH
+DEBUG   Running command: where chrome
+DEBUG   Output: ""
+DEBUG   chrome not found in PATH
+DEBUG   chrome has not been discovered in the system
+DEBUG   Reading metadata from https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions-with-downloads.json
+DEBUG   Required browser: chrome 116.0.5845.49
+DEBUG   Downloading chrome 116.0.5845.49 from https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.49/win64/chrome-win64.zip
+DEBUG   chrome 116.0.5845.49 has been downloaded at C:\Users\boni\.cache\selenium\chrome\win64\116.0.5845.49\chrome.exe
+DEBUG   Reading metadata from https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json
+DEBUG   Required driver: chromedriver 116.0.5845.49
+DEBUG   Driver URL: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.49/win64/chromedriver-win64.zip
+INFO    Driver path: C:\Users\boni\.cache\selenium\chromedriver\win64\116.0.5845.49\chromedriver.exe
+INFO    Browser path: C:\Users\boni\.cache\selenium\chrome\win64\116.0.5845.49\chrome.exe
+```
+
+### Troubleshooting in the bindings
+
+If you want to get the Selenium Manager debugging information when using the Selenium bindings languages, you can enable it as follows (in Java):
+
+```
+Arrays.stream(Logger.getLogger("").getHandlers()).forEach(handler -> {
+    handler.setLevel(Level.FINE);
+});
+Logger.getLogger(SeleniumManager.class.getName()).setLevel(Level.FINE);
+```
+
+For the equivalence in other bindings languages, visit the [Selenium troubleshooting page](https://www.selenium.dev/documentation/webdriver/troubleshooting/).
+
+### Next steps
+
+The following features to be implemented in Selenium Manager will continue the automated browser management mechanism, this time for **Firefox** and **Edge**. Stay tuned!
```

Comment:
We can also link to https://github.com/orgs/SeleniumHQ/projects/5 if you want to show that we have a bunch of other things in mind.

Reply:
Done.